### PR TITLE
Improve pr-check

### PR DESCRIPTION
**Pull request by mihaic**
Remove possible left-over coverage files.

Point contributors to pr-check more prominently.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -39,9 +39,11 @@ generate pull requests to submit your code for inclusion in the project.
 .. _GitHub help for collaborating on projects using issues and pull requests:
    https://help.github.com/categories/collaborating-on-projects-using-issues-and-pull-requests/
 
-All pull requests will be automatically tested as described below, so make sure
-you test them yourself before submitting them::
+All pull requests will be automatically tested as described below using the
+``pr-check.sh`` script, so make sure you test them yourself before submitting
+them::
 
+  # do not run this if using Anaconda; instead, look at pr-check.sh
```
Comment (yidawang) [inline CONTRIBUTING.rst]: This statement is not clear to me. Look at `pr-check.sh` for what?

Comment (mihaic) [inline CONTRIBUTING.rst]: Clarified.

Comment (yidawang) [inline CONTRIBUTING.rst]: Can you also elaborate why running this via Anaconda is not recommended? Using Anaconda one can directly run `./pr-check.sh` to test out before making the PR.

Comment (mihaic) [inline CONTRIBUTING.rst]: Done.

**Comment by mihaic** [comment]
Jenkins, retest this please.

**Pull request merged**